### PR TITLE
[macOS] Replace '+' from java version folder to '-'

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
# Description
In scope of this pull request we edit logic to replace '+' in java version path to '-'.

## Check list
- [ ] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by maxim-lobanov — APPROVED** [review]

**Review by al-cheb — APPROVED** [review]

**Review by dibir-magomedsaygitov — APPROVED** [review]

**Review by nikita-bykov — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

File: images/macos/software-report/SoftwareReport.Java.psm1 [hunk 1]
```diff
@@ -10,10 +10,11 @@ function Get-JavaVersions {
         $javaPath = $_.Value
         # Take semver from the java path
         $version = $javaPath.split('/')[5]
+        $fullVersion = $version.Replace('-', '+')
```
Comment (dibir-magomedsaygitov) [inline images/macos/software-report/SoftwareReport.Java.psm1:13]: I think that it should be like:
$fullVersion = $version.Replace('+', '-')

Comment (dibir-magomedsaygitov) [reply images/macos/software-report/SoftwareReport.Java.psm1:13]: Or do we want to state `+` in docs?

**Review by MaksimZhukov — APPROVED** [review]

File: images/macos/software-report/SoftwareReport.Java.psm1 (hunk 1 above)
Comment (dmitry-shibanov) [reply images/macos/software-report/SoftwareReport.Java.psm1:13]: In the path it should be like `8.0.282-8`, but for software report version should be as `8.0.282+8`

**Pull request merged**